install_files/ansible-base/roles/build-securedrop-app-code-deb-pkg/files/usr.sbin.apache2

```diff
@@ -51,6 +51,9 @@
   /etc/apache2/mods-available/rewrite.load r,
```

Comment:
Perhaps a nit: From the diff, it seems like there are a lof of changes here. On closer inspection, some entries are simply reordered. Would it be possible to keep the diff as minimal as possible here to more easily understand the changes needed for the new build process?

Reply:
@emkll I ordered them because I found changing the unsorted rules error-prone, and thought having related rules grouped would make changes (and review :slightly_smiling_face:) easier. The diff is still fairly sizable without reordering, unfortunately.